test/collection-facets/more-facets-content.test.ts

```diff
@@ -126,21 +127,52 @@ describe('More facets content', () => {
     expect(bucket?.count).to.equal(5);
   });
 
-  it('page number clicked event', async () => {
```

Comment:
Q: did we lost page number clicked coverage? this may be the only test that page clicking has..